### PR TITLE
Log utils

### PR DESCRIPTION
**Pull request by kamil-triscik**

### CODE REVIEW

File: logging-integration-tests/src/main/java/cz/muni/pa036/logging/log/LogLevel.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    OFF,
+    TRACE,
+    WARN;
+
+    /**
+     * Method transform string level object to LogLevel object.
+     * @param level string representation of level.
+     * @return LogLevel representation.
+     */
+    public static LogLevel getLevel(String level) {
```
Comment (f-gdovin) [inline logging-integration-tests/src/main/java/cz/muni/pa036/logging/log/LogLevel.java]: I believe you can use "LogLevel.valueOf(level.toUpperCase())", which is provided by Enum class

Comment (f-gdovin) [inline logging-integration-tests/src/main/java/cz/muni/pa036/logging/log/LogLevel.java]: OR this, which also uses Enum.values

public static LogLevel fromString(String level) {
    for (LogLevel logLevel : LogLevel.values()) {
      if (logLevel.text.equalsIgnoreCase(level)) {
        return logLevel;
      }
    }
    return null;
  }

File: logging-persistence/src/main/java/cz/muni/pa036/logging/configuration/DBConfig.java [hunk 2]
```diff
@@ -76,13 +75,21 @@ public LoadTimeWeaver instrumentationLoadTimeWeaver() {
...
+			dataSource.setUrl(properties.getProperty("jdbc.url"));
+			dataSource.setUsername(properties.getProperty("jdbc.user"));
+			dataSource.setPassword(properties.getProperty("jdbc.pass"));
 
-		return dataSource;
+			return dataSource;
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+		return null;//tak vyhodi chybu, no nespusti, kaslem na to
```
Comment (f-gdovin) [inline logging-persistence/src/main/java/cz/muni/pa036/logging/configuration/DBConfig.java]: lol :D

File: logging-persistence/src/main/resources/database.properties [hunk 3]
```diff
@@ -1,4 +1,4 @@
--jdbc.driver=org.postgresql.Driver
--jdbc.url=jdbc:postgresql://localhost:5432/pa036
--jdbc.user=postgres
--jdbc.pass=postgres 
+jdbc.driver=org.postgresql.Driver
+jdbc.url=jdbc:postgresql://localhost:5432/pa036
+jdbc.user=kamil
```
Comment (f-gdovin) [inline logging-persistence/src/main/resources/database.properties]: come on, we all have postgres/postgres, can you create that user as well for this DB?

Comment (kamil-triscik) [inline logging-persistence/src/main/resources/database.properties]: just my mistake, should not be committed

File: logging-persistence/src/main/java/cz/muni/pa036/logging/configuration/DBConfig.java (hunk 2 above)
Comment (kamil-triscik) [inline logging-persistence/src/main/java/cz/muni/pa036/logging/configuration/DBConfig.java]: not part of project

File: logging-persistence/src/main/resources/database.properties (hunk 3 above)
Comment (kamil-triscik) [inline logging-persistence/src/main/resources/database.properties]: fixed

File: logging-integration-tests/src/main/java/cz/muni/pa036/logging/log/LogLevel.java (hunk 1 above)
Comment (kamil-triscik) [inline logging-integration-tests/src/main/java/cz/muni/pa036/logging/log/LogLevel.java]: Custom **LogLevel** replaced by **ch.qos.logback.classic.Level**

**Pull request merged**